gradle/shared-with-buildSrc/mirrors.settings.gradle.kts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+val originalUrls: Map<String, String> = mapOf(
+    "jcenter" to "https://jcenter.bintray.com/",
+    "mavencentral" to "https://repo.maven.apache.org/maven2/",
+    "google" to "https://dl.google.com/dl/android/maven2/",
+    "gradle" to "https://repo.gradle.org/gradle/repo",
```

Comment:
Sometimes the repo has a `/` at the end and sometimes it doesn't. Is this intentional?

Reply:
I don't think so... I copied from here: https://github.com/gradle/gradle/blob/6fb2b8496a86adacf701503b5d52bc657788b716/subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/RepoScriptBlockUtil.groovy#L32

I think people just don't notice this issue when they were adding urls into this class...